ODPi-Runtime.md

```diff
@@ -273,7 +473,7 @@ Best practices for compatible apps to be portable and operator friendly:
 
 -   Applications SHOULD NOT assume that an Oracle JRE is being used.
 
--   Applciations SHOULD include both Microsoft batch or PowerShell as well as Unix-compatible shell scripts.
+-   Applications SHOULD include both Microsoft batch or PowerShell as well as Unix-compatible shell scripts.
```

Comment:
I think it would good for us to separate out "clean things up" changes from the "this adds HCFS ones" - ideally tying back the former into their own JIRAs. Yes, it's a PITA, but when we look at this a year later we won't wonder what we were thinking ;-).